src/include/configuration/configuration.h

```diff
@@ -31,6 +31,16 @@ DECLARE_uint64(max_connections);
 // Socket family
 DECLARE_string(socket_family);
 
+// Added for SSL only begins
+
+// Peloton private key file
+DECLARE_string(private_key_file);
+
+// Peloton certificate file
+DECLARE_string(certificate_file);
+
+// Added for SSL only ends
```

Comment:
Are "// Added for SSL only begins" and "// Added for SSL only ends" extra comments without code?